### ISSUE TITLE
Quick Log In

### ISSUE THREAD

**Issue by BreadSpam** [cs6413110/Pixel-Tanks#25]
Since we added html log in, we can finally finish this right?

make it so you can save ur password on the site, so when u go back, u can sign in faster

maybe also add a Google sign in option using ur school email, if that's necessary for quick logs

**Comment by LostKing73** [comment]
also like email verification for accounts

**Comment by cs6413110** [comment]
google don't like to save passwords to the data url but I can try

**Comment by 3foe** [comment]
I think it's a good idea I really do, but if google can track it doesn't that mean securely can because it has access to all passwords in permissions, so I think no

**Comment by LostKing73** [comment]
k, 3-1 then yall, watch out

**Comment by 3foe** [comment]
that was everyones vote, **before** they saw my concern. And if you paid attention to what I warned @cs6413110 about the principal is going to talk to and try to punish everyone on the GitHub. My dad made sure of that. so this gives them a way to do so. (that is also why I am using gamer names)

**Comment by LostKing73** [comment]
k.

**Comment by 3foe** [comment]
yeah but school can track and take away chromebooks...

**Comment by DarkMemeGod411** [comment]
yeah, remember what happened to us in 8th grade with us all getting called down to office?

**Comment by DarkMemeGod411** [comment]
> that was everyones vote, **before** they saw my concern. And if you paid attention to what I warned @cs6413110 about the principal is going to talk to and try to punish everyone on the GitHub. My dad made sure of that. so this gives them a way to do so. (that is also why I am using gamer names)

Your dad told the school about us. 3foe?

**Comment by DarkMemeGod411** [comment]
Being on the schools watchlist, Sounds Splendid

**Comment by LostKing73** [comment]
lol, aaron make it work on phones. ez fix

**Comment by LostKing73** [comment]
k so no negatives? lets go

**Comment by DarkMemeGod411** [comment]
would this be for quick log in or all log ins?

**Comment by LostKing73** [comment]
wdym all logins

**Comment by DarkMemeGod411** [comment]
like when you login with typing

**Comment by DarkMemeGod411** [comment]
would all need it or only for fast logins

**Comment by LostKing73** [comment]
bruh speak english

**Comment by LostKing73** [comment]
what else? theres only one login for pickle tanks

**Comment by DarkMemeGod411** [comment]
the current login system, typing it all yourself, "the quick login" is different

**Comment by DarkMemeGod411** [comment]
i am saying will i need a phone to log in normally or not?

**Comment by LostKing73** [comment]
no. i meant make the game work on the phone dum

**Comment by DarkMemeGod411** [comment]
oh

**Comment by BreadSpam** [comment]
Can't get watchlisted again ez lol

**Comment by DarkMemeGod411** [comment]
how?

**Comment by DarkMemeGod411** [comment]
> Can't get watchlisted again ez lol

where you watchlisted?
